tensornetwork/backends/abstract_backend.py

```diff
@@ -1044,3 +1044,5 @@ def eps(self, dtype: Type[np.number]) -> float:
 
     raise NotImplementedError(
         f"Backend {self.name} has not implemented eps.")
+  def initialize_orthogonal_tensor_wrt_pivot(self,shape=Sequence[int],dtype:Optional[Type[np.number]]=None,pivot_axis:int=-1,seed=Optional[int]=None,backend: Optional[Union[Text, AbstractBackend]] = None,non_negative_diagonal: bool = False):->Tensor
```

Comment:
pls break this line (our linter should have actually complained about this ...)

Reply:
@mganahl done.